### DISCUSSION TITLE
Need help with JSon wall swich

### DISCUSSION THREAD

**Discussion by orson32** [RavenSystem/esp-homekit-devices#2127] [MEPLHAA Scripts]
Please help, I have not been able to create the correct JSON
     {"c":{"io”:[[[12,13,2],2],[[2],6],[[16,5],6,0,1]],”l”:2,”b”:[[2,16]]},”a”:[{“0”:{“r”:[[12]]},”1”:{“r”:[[12,1]]},”b”:[[16,0]]},{“0”:{“r”:[[13]]},”1”:{“r”:[[13,1]]},”b”:[[5,0]]}]}

![IMG_3112](https://github.com/RavenSystem/esp-homekit-devices/assets/81135446/d3ae71d1-f7db-4c75-8606-7f75953a5269)

**Comment by xrust83** [comment]
> Please help, I have not been able to create the correct JSON {"c":{"io”:[[[12,13,2],2],[[2],6],[[16,5],6,0,1]],”l”:2,”b”:[[2,16]]},”a”:[{“0”:{“r”:[[12]]},”1”:{“r”:[[12,1]]},”b”:[[16,0]]},{“0”:{“r”:[[13]]},”1”:{“r”:[[13,1]]},”b”:[[5,0]]}]}
> 
> ![IMG_3112](https://user-images.githubusercontent.com/81135446/272471515-d3ae71d1-f7db-4c75-8606-7f75953a5269.jpeg)

You have a problem in quotes, what looks the same to the eye to the computer is different symbols. type quotes in eng i directly in your browser. Do not transfer from word or other text editors. look carefully at the quote before “io” and after “io”

**Reply by PhaseAngle** [reply]
Turn off smart quotes

**Comment by RavenSystem** [comment]
```json
{"c":{"io":[[[12,13,2],2],[[16,5],6]],"c":1,"l":2,"i":0},"a":[{"0":{"r":[[12]]},"1":{"r":[[12,1]]},"b":[[5]]},{"0":{"r":[[13]]},"1":{"r":[[13,1]]},"b":[[16]]}]}
```
